docs/draft/EntandoFreeMarkerTags/FreemarkerJACMSTags.adoc

```diff
@@ -21,12 +21,12 @@ a|
 Attributes
 ++++++++++
 
-|Attribute: contentId
+|Attribute: contentID
```

Comment:
@dale1391 we can't change these. These are actual values used in the code and they need to match. The ones in the descriptions are ok though.